ADApp/pluginSrc/NDPluginDriver.cpp

```diff
@@ -357,7 +361,11 @@ asynStatus NDPluginDriver::writeInt32(asynUser *pasynUser, epicsInt32 value)
         this->lock();
     } else if (function == NDPluginDriverQueueSize) {
         newQueueSize_ = value;
-     } else {
+    } else if (function == NDPluginDriverProcessPlugin) {
+        if (pInputArray_) {
+            driverCallback(pasynUserSelf, pInputArray_);
+        }
```

Comment:
Should we return an error status here if a pInputArray_ is not available instead of silently carrying on?